### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by 1hko** [1hko/elm-truth-table#1]
We need tests for this library but currently _computation_ of the result and _rendering_ of the result is tangled together.  Separating the two will enable the library to be used in more general ways too.

I'm still new to Elm and I don't know how to write tests for it yet. Any help in this area would be appreciated.

**Comment by janwirth** [comment]
I agree that the computation of the results and the rendering should be disentangled. Have you considered using unit types rather than strings for variables?

**Comment by 1hko** [comment]
> Have you considered using unit types rather than strings for variables?

Can you explain a bit more please?

**Comment by 1hko** [comment]
> Have you considered using unit types rather than strings for variables?

Can you explain a bit more please?

I'm still new to Elm and I just reached for the quickest thing that could make this work. I'm open to any recommendations 😄
